src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -166,7 +166,7 @@ public static JsonObject of(String k1, Object v1, String k2, Object v2, String k
    */
   public static JsonObject of(String k1, Object v1, String k2, Object v2, String k3, Object v3,
                               String k4, Object v4) {
-    JsonObject obj = new JsonObject(new LinkedHashMap<>(4));
+    JsonObject obj = new JsonObject(new LinkedHashMap<>(7));
```

Comment:
```suggestion
    JsonObject obj = new JsonObject(new LinkedHashMap<>(6));
```
https://guava.dev/releases/21.0/api/docs/src-html/com/google/common/collect/Maps.html#line.291